b2b-sample/guardio-insurance-sample-application/components/settingsComponents/homeComponet/homeComponent.js

```diff
@@ -49,18 +50,16 @@ export default function HomeComponent(props) {
                 </div>
             </Panel>
             {
-                me == null ?
-                    <Panel bordered>
+                me == null
+                    ? <Panel bordered>
                         <div>Add the user attributes in created the application to display user details</div>
                     </Panel>
-                    :
```

Comment:
`Add the user attributes in the created application to display user details`

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/00fcf2af0c800b9bfc234e95534cb746916ccd07